src/sentry/static/sentry/app/components/stream/processingIssueList.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import {isEqual} from 'lodash';
+import PropTypes from 'prop-types';
+
+import {Client} from 'app/api';
+import {logAjaxError} from 'app/utils/logging';
+import {fetchProcessingIssues} from 'app/actionCreators/processingIssues';
+import ProcessingIssueHint from 'app/components/stream/processingIssueHint';
+import SentryTypes from 'app/sentryTypes';
+
+class ProcessingIssueList extends React.Component {
+  static propTypes = {
+    organization: SentryTypes.Organization.isRequired,
+    projectIds: PropTypes.array,
+    showProject: PropTypes.bool,
+  };
+
+  static defaultProps = {
+    showProject: false,
+  };
+
+  constructor(props) {
+    super(props);
+    this.api = new Client();
+    this.state = {
+      loading: true,
+      issues: null,
```

Comment:
You could initialize this with `[]` so issues is always an array, and can drop lines 67-69 in the render statement